### DISCUSSION TITLE
Restricting only a subset of pages

### DISCUSSION THREAD

**Discussion by d3cxxxx** [oidc-wp/openid-connect-generic#271] [Q&A]
Can I use this plugin to restrict only a subset of pages in my wordpress site and leave the rest of the site open to public?

**Comment by timnolte** [comment]
> Can I use this plugin to restrict only a subset of pages in my wordpress site and leave the rest of the site open to public?

So this plugin is primarily designed to provide OpenID Connect Single Sign On for the WordPress dashboard, there is an optional setting that will also put the entire site behind the SSO signing requirement. There isn't currently the capability to restrict specific pages of your frontend behind SSO. You could use a plugin such as Members to create a public page restriction to a logged in user and then provide OpenID Connect as the means for logging in your users.
